docs/source/models/cause.rst

```diff
@@ -16,6 +16,62 @@ What is a cause?
 The structure of a cause model
 ------------------------------
 
+Model impact 
```

Comment:
Let's call this "Cause model impact" to make to completely clear that it is about the impact of a cause model on the demographic model.

Reply:
changed, thanks